styles/layout.css

```diff
@@ -200,6 +200,12 @@ footer small,
   padding: 0 0.5em;
   display: table-cell;
 }
+.quick-links {
+  padding: 0.5em 0;
```

Comment:
Here and in `layout.less`, I'm curious about why padding is used instead of margin (mind you, I'm totally not an expert). I realize that we want some space top and bottom, but padding -- even a half em top and bottom -- adds empty space and is not collapsible.

I don't think the call-out box should be a concern; at least not the one for newbies. Except on the test site, newbies with fewer than 100 pages proofed are extremely unlikely to ever see the links, as they won't have proofed enough pages to have access to Post-Processing.

Reply:
using margin now indirectly through ul/li tags now.